### PR TITLE
Added ari-web.xyz dark subdomains

### PR DESCRIPTION
**Pull request by TruncatedDinosour**

### CODE REVIEW

**Comment by TruncatedDinosour** [comment]
Idk if it's supported so I was scared to do it, if globbing is supported I should replace my subdomains with

```regex
ari-web.xyz
*.ari-web.xyz
*.*.ari-web.xyz
```

or if regex

```regex
(.*\.)?ari-web\.xyz
```

File: src/config/dark-sites.config [hunk 1]
```diff
@@ -51,6 +51,7 @@ ara-ara-ufufu.herokuapp.com
 arche-dev.codeberg.page
 archive.ragtag.moe
 argoto.snaz.in
+ari-web.xyz
```
Comment (Gusted) [inline src/config/dark-sites.config:54]: You can remove the other entries and only leave this one in.

Comment (TruncatedDinosour) [reply src/config/dark-sites.config:54]: @Gusted Ah thanks, wait

Comment (TruncatedDinosour) [reply src/config/dark-sites.config:54]: done

**Review by Gusted — APPROVED** [review]

**Pull request merged**